app/scripts/modules/amazon/src/serverGroup/details/sections/SecurityGroupsDetailsSection.tsx

```diff
@@ -59,12 +59,12 @@ export class SecurityGroupsDetailsSection extends React.Component<
     const { securityGroups } = this.state;
 
     return (
-      <CollapsibleSection heading="Security Groups">
+      <CollapsibleSection heading={FirewallLabels.get('Firewalls')}>
         <ul>
           {sortBy(securityGroups, 'name').map(securityGroup => (
             <li key={securityGroup.name}>
               <UISref
-                to="^.securityGroupDetails"
+                to="^.firewallDetails"
```

Comment:
what's the story behind updating this one but not all the other links to `^.securityGroupDetails`?

Reply:
Thanks for catching - I had started to convert the routes but forgot about it. I've updated the PR to convert all the routes.